src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -48,7 +48,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.8" />
-    <PackageReference Include="System.Linq.Dynamic.Core" Version="1.0.8" />
+    <PackageReference Include="System.Linq.Dynamic.Core" Version="1.0.8.11" />
```

Comment:
The 1.0.8 version did not work on mono/.NET Core due to this bug: https://github.com/StefH/System.Linq.Dynamic.Core/issues/157. So updated to latest.